CONTRIBUTING.md

```diff
@@ -323,9 +325,13 @@ For all OS types perform the following:
 #### RHEL 7/8
 
 1. Run `pyinstaller -F ./pkg_bin/conjur`. Once this is run, a `dist` folder will be created with the executable in it.
```

Comment:
I know this wasn't your doing but I feel like it's not clear that its two different machines. Can you update this first sentence to:
```
Run `pyinstaller -F ./pkg_bin/conjur` on the different RHEL 7 and RHEL 8 machines.
```